HACKING.md

```diff
@@ -0,0 +1,90 @@
+# Hacking on Libreport
+
+Here's where to get the code:
+
+    $ git clone https://github.com/abrt/libreport.git
+    $ cd libreport/
+
+The remainder of the commands assume you're in the top level of the
```

Comment:
assumes

Reply:
It’s definitely also used as a plural noun as well, but not sure what prescriptivists have to say about that.

Reply:
I agree with Mirek, s/assume/assume**s**/

```markdown
Collective nouns usually take **is**, but you can use **are** if you need to
emphasize the individuals who belong to the group.
```
[https://www.grammarly.com/blog/is-vs-are/](https://www.grammarly.com/blog/is-vs-are/)

Reply:
Updated @mkutlak